src/features/DebugSession.ts

```diff
@@ -386,3 +386,130 @@ export class PickPSHostProcessFeature implements IFeature {
         }
     }
 }
+
+interface IRunspaceItem extends vscode.QuickPickItem {
+    id: string;	// payload for the QuickPick UI
+}
+
+interface IRunspace {
+    id: string;
+    name: string;
+    availability: string;
+}
+
+export const GetRunspaceRequestType =
+    new RequestType<any, IGetRunspaceResponseBody, string, void>("powerShell/getRunspace");
+
+interface IGetRunspaceResponseBody {
+    runspaces: IRunspace[];
+}
+
+export class PickRunspaceFeature implements IFeature {
+
+    private command: vscode.Disposable;
+    private languageClient: LanguageClient;
+    private waitingForClientToken: vscode.CancellationTokenSource;
+    private getLanguageClientResolve: (value?: LanguageClient | Thenable<LanguageClient>) => void;
+
+    constructor() {
+
```

Comment:
nit: you can probably get rid of this newline

Reply:
Done